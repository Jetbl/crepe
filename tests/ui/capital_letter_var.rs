// This test checks behavior that identifiers starting with 'A'..'Z' are not
// treated as Datalog variables.

use crepe::crepe;

crepe! {
    struct Ok(i32);
    Ok(Not_a_variable) <- Ok(Not_a_variable);
}

fn main() {}
