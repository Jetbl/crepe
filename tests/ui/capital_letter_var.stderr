error[E0425]: cannot find value `Not_a_variable` in this scope
 --> tests/ui/capital_letter_var.rs:8:30
  |
8 |     Ok(Not_a_variable) <- Ok(Not_a_variable);
  |                              ^^^^^^^^^^^^^^ not found in this scope

error[E0425]: cannot find value `Not_a_variable` in this scope
 --> tests/ui/capital_letter_var.rs:8:8
  |
8 |     Ok(Not_a_variable) <- Ok(Not_a_variable);
  |        ^^^^^^^^^^^^^^ not found in this scope
